### PR TITLE
chore(release): 1.7.0

### PR DESCRIPTION
**Pull request by RitvikKapila**
_Issue #, if available:_

_Description of changes:_
Update changelog for release version 1.7.0

_Squash/merge commit message, if applicable:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts [hunk 1]
```diff
@@ -18,7 +18,7 @@ var props = Properties().apply {
 var dafnyVersion = props.getProperty("dafnyVersion")
 
 group = "software.amazon.cryptography"
-version = "1.6.0-SNAPSHOT"
+version = "1.7.0-SNAPSHOT"
```
Comment (josecorella) [inline AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:21]:
```suggestion
version = "1.7.0"
```

File: TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts
Comment (josecorella) [inline TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:1]: the releaserc should have bumped tline 72 as well to 1.7.0

**Review by josecorella — CHANGES_REQUESTED** [review]

File: TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts [hunk 2]
```diff
@@ -20,7 +20,7 @@ var props = Properties().apply {
 var dafnyVersion = props.getProperty("dafnyVersion")
 
 group = "software.amazon.cryptography"
-version = "1.6.0-SNAPSHOT"
+version = "1.7.0"
```
Comment (josecorella) [inline TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:23]: doesn't really matter but I think for testvectors this always remains a snapshot

**Review by josecorella — APPROVED** [review]

File: TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts (hunk 2 above)
Comment (RitvikKapila) [reply TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:23]: I'll fix this before finishing the MCM

Comment (RitvikKapila) [reply TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:23]: Thanks for pointing it out.

**Pull request merged**